vitest.config.ts

```diff
@@ -3,24 +3,26 @@ import { defineConfig } from "vitest/config";
 
 export default defineConfig({
 	test: {
-		include: ["src/**/*.test.ts"],
+		globals: true,
+		environment: "node",
+		include: ["test/unit/**/*.test.ts", "test/integration/**/*.test.ts"],
 		exclude: [
+			"test/integration/**",
 			"**/node_modules/**",
-			"**/dist/**",
-			"**/build/**",
 			"**/out/**",
-			"**/src/test/**",
-			"src/test/**",
-			"./src/test/**",
+			"**/*.d.ts",
 		],
-		environment: "node",
+		pool: "threads",
+		fileParallelism: true,
 		coverage: {
 			provider: "v8",
 		},
 	},
 	resolve: {
 		alias: {
-			vscode: path.resolve(__dirname, "src/__mocks__/vscode.runtime.ts"),
+			"@": path.resolve(__dirname, "src"),
+			"@test": path.resolve(__dirname, "test"),
```

Comment:
Do you feel strongly about the `@` and `@test` aliases?  I like to see the real relative path to the import, partially because it makes it easier to jump to or complete the file when I am not feeling like launching the language server and partially because they feel too magical for no real benefit (at least from what I have seen).